compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -164,8 +165,12 @@ fun HeaderCell(
 fun TableHeaderRow(
     tableModel: TableModel,
     horizontalScrollState: ScrollState,
-    selectionState: SelectionState
+    selectionState: SelectionState,
+    isFirstTable: Boolean
```

Comment:
Maybe `isFirstTable`  can be a logic introduce in the mapper when creating _TableModel_, for me makes sense to be a propperty of the table itslef. Maybe can be something like _tableHeight_: Int and can be 0 to first and 16 (converted) for the rest. With this we can test it in the mapper.  What do you think?